docs/01-basics/LZ-01-06.adoc

```diff
@@ -0,0 +1,37 @@
+
+// tag::DE[]
+[[LZ-01-06]]
+==== LZ 01-06: Bedeutung und Entwurf von Datenmodellen (R2)
+
+Softwarearchitekt:innen verstehen die Bedeutung von Datenmodellen für
+die Architektur.  Sie
+
+* können Datenmodelle identifizieren, die maßgeblichen Einfkuss auf
+  die Architektur haben. (R2)
+* können solche Datenmodelle systematisch entwerfen. (R2)
+* verstehen den Unterschied zwischen Produkten und Summen in der
+  Datenmodellierung. (R1)
```

Comment:
Sind Produkt und Summe in der Community akzeptierte Begriffe? Die Begriffe "zusammengesetzte Daten" und "gemischte Daten" finde ich sprechender, allgemein bekannt sind die aber sich auch nicht. Warum ist das R1 wenn im Titel nur R2 steht?

Reply:
Warum: Nachlässigkeit ... :-)

Die Begriff sind zumindest gut Internet-suchbar.

Reply:
Wäre das nicht Einträge im Glossar wert? Da stehe Dinge wie Visitor drin, dann sollte auch Produkt und Summe rein.

Reply:
ja Stefan, das ist sicher einen Glossary-Entry wert.